src/index.ts

```diff
@@ -201,6 +201,11 @@ const moduleTransform: Omit<Transformer, 'getCacheKey'> = {
                 break;
         }
 
+        if (preProcessorsConfig.inject) {
+            const styleInject = require('style-inject');
+            styleInject(textCSS);
+        }
+
```

Comment:
This code should be inside exported module(not in transformer code). Something like this:

```js
if (preProcessorsConfig.injectIntoJSDOM) {
  const moduleCode = moduleTemplate.replace('%s', JSON.stringify(parser.getCSSSelectors(textCSS)));
  return `
  const style = document.createElement('style');
  style.styleSheet.cssText = `${textCSS}`;
  document.head.appendChild(style);
  ${moduleCode}
`
}
```

Reply:
Good catch. We might need to strip indentation here because of the literal. Are you okay with using a lib for that or we leave it with the stray indent ?

Reply:
Simple join is quite good solution:

```js
if (preProcessorsConfig.injectIntoJSDOM) {
  const moduleCode = moduleTemplate.replace('%s', JSON.stringify(parser.getCSSSelectors(textCSS)));
  const textCssEscaped = textCSS.replace(/`/g, '\`');

  return [
    'const style = document.createElement(\'style\');',
    `style.styleSheet.cssText = \`${textCssEscaped}\`;`,
    'document.head.appendChild(style);',
    moduleCode
  ].join('\n');
}
```